conformance/base/manifests.yaml

```diff
@@ -223,6 +223,53 @@ spec:
 ---
 apiVersion: v1
 kind: Service
+metadata:
+  name: infra-backend-delay
+  namespace: gateway-conformance-infra
+spec:
+  selector:
+    app: infra-backend-delay
+  ports:
+  - protocol: HTTP
+    port: 8000
+    targetPort: 80
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: infra-backend-delay
+  namespace: gateway-conformance-infra
+  labels:
+    app: infra-backend-delay
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: infra-backend-delay
+  template:
+    metadata:
+      labels:
+        app: infra-backend-delay
+    spec:
+      containers:
+      - name: infra-backend-delay
+        # TODO: can we enhance echo server to support a delayed response and use it here instead of httpbin?
+        image: docker.io/kong/httpbin
```

Comment:
We already use Istoi's "echo" in mesh tests which support this - maybe can use that

Reply:
Great! I didn't know the Istio echo server supported delay.

Is this the Istio echo image you mean? `image: gcr.io/k8s-staging-gateway-api/echo-server:v20230505-v0.7.0-rc1-10-g497e67da`

Then we just call it with a path like this: `/?delay=1s`?